packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx

```diff
@@ -61,8 +89,13 @@ const TradingHubOnboarding = ({ is_dark_mode, setIsOnboardingVisited }) => {
                         icon={is_dark_mode ? 'IcAppstoreTradingHubOnboardingDark' : 'IcAppstoreTradingHubOnboarding'}
                         size={20}
                         onClick={() => {
-                            history.push(routes.onboarding);
-                            setIsOnboardingVisited(false);
+                            // TODO: implement if else function to show traders hub onboarding
+                            if (data?.length > 0) {
```

Comment:
@aizad-deriv need to check the `feature_flag` here
```suggestion
                            if (data?.length > 0 && is_wallet_enabled) {
```